### ISSUE TITLE
Cannot find portfolio entries

### ISSUE THREAD

**Issue by igllnd** [Somrat37/somrat#24]
I got it running with no issues so far but I really can't figure out how to actually work with the portfolio part. Adding new *.md files to the portfolio directory and having them show up works fine but where do you see all the text that is written in the project.md file? As far as I can see you only get a picture in the portfolio section and when you click on it you get a gallery. But since there is text in the .md file I assume that you should also be able to land on a project page for that portfolio entry. But how?

I just saw that your project is named "archived" but I still hope you could help me out here. Thanks!

**Comment by siranishkd** [comment]
Indeed, I looked at his hosted website, the portfolio doesn't link to a new page. I hope you could help us out.
